requirements.txt

```diff
@@ -18,6 +18,7 @@ google-cloud-logging==3.12.1
 google-cloud-secret-manager==2.25.0
 google-cloud-storage==3.4.1
 google-auth==2.42.1
+google-genai==1.48.0
```

Comment:
1.49.0 was just released! Can update here or let dependabot update it if you want.

Reply:
I guess it's fine to leave this as is, and we'll see if Dependabot picks it up as expected.